pkg/cmd/server/kubernetes/node/options/options.go

```diff
@@ -82,6 +82,10 @@ func Build(options configapi.NodeConfig) (*kubeletoptions.KubeletServer, error)
 	server.RemoteImageEndpoint = options.DockerConfig.DockerShimSocket
 	server.DockershimRootDirectory = options.DockerConfig.DockershimRootDirectory
 
+	// TODO: check/warn/fail in setup instead?
+	// allows kubelet to continue to start in swap environments
+	server.FailSwapOn = false
```

Comment:
This shouldn't be defaulting on when we ship 1.9

Reply:
@liggitt this is to fix our test infra because we run CI with swap on? For cluster up I think I plumbed it inside cluster up.

Reply:
You can't fix test infra from here.

Reply:
yeah, I think we will need to fix ansible by passing a flag to kubelet?

Reply:
confirmed with @sdodson, ansible disables swap. will back this out as a follow up and limit to just `cluster up`